server/plugin/command.go

```diff
@@ -6,23 +6,26 @@ import (
 	"strings"
 	"unicode"
 
+	"github.com/google/go-github/v41/github"
 	"github.com/mattermost/mattermost-plugin-api/experimental/command"
 	"github.com/mattermost/mattermost-server/v6/model"
 	"github.com/mattermost/mattermost-server/v6/plugin"
 	"github.com/pkg/errors"
 )
 
 const (
-	featureIssueCreation = "issue_creations"
-	featureIssues        = "issues"
-	featurePulls         = "pulls"
-	featurePullsMerged   = "pulls_merged"
-	featurePushes        = "pushes"
-	featureCreates       = "creates"
-	featureDeletes       = "deletes"
-	featureIssueComments = "issue_comments"
-	featurePullReviews   = "pull_reviews"
-	featureStars         = "stars"
+	featureIssueCreation          = "issue_creations"
+	featureIssues                 = "issues"
+	featurePulls                  = "pulls"
+	featurePullsMerged            = "pulls_merged"
+	featurePushes                 = "pushes"
+	featureCreates                = "creates"
+	featureDeletes                = "deletes"
+	featureIssueComments          = "issue_comments"
+	featurePullReviews            = "pull_reviews"
+	featureStars                  = "stars"
+	NoWebhookFoundMsg             = "\nNo webhook was found for this repository or organization. To create one, enter the following slash command `/github setup webhook`"
```

Comment:
```suggestion
	NoWebhookFoundMsg             = "\nNo webhook was found for this repository or organization. To create one, run the following slash command `/github setup webhook`"
```

Reply:
or `execute` maybe?

Reply:
The msg is given by mike https://mm.brightscout.com/freelancers/pl/wgjacqcoz3nu8n4mdgndedwhte